_routes/comments.md

```diff
@@ -104,3 +126,10 @@ POST /api/v1/posts/:post_guid/comments
   "body": "Can I use these for my own website?"
 }
 ~~~
+
+### Errors
+
+Status code  |  Error reason
+-|-
+404  |  Post with provided guid was not found
+422  |  Client is not allowed to comment (user is ignored)
```

Comment:
A 422 may also be a generic validation error, so not sure if we need to do something about htat here.

Reply:
Maybe then `The comment is not allowed`? This way we don't specify that the user is to blame, but that the comment might be not allowed for other reasons.

Reply:
422 is hard to word right, yeah. Perhaps just somehting like "Couldn't accept or process X"?

Reply:
So `Couldn't accept  or process the comment`? Is it okay to put `user is ignored` in brackets as a possible reason for that?

There is also a similar one for conversations: https://github.com/diaspora/api-documentation/pull/11#discussion_r194099974, where I can put `Couldn't accept or process the conversation`, I think?

Reply:
Actually I just noticed that for likes I put two different lines for `422` one for ignored and one for failed validation. To be consistent I can put two different lines here too.

Reply:
I think I'd rather prefer one line per status code only :)